File: Sample Code/DeBiese.Authorization/DeBiese.Authorization.NG/app/models/PostData.js
'use strict';
var DeBiese;
(function (DeBiese) {
    var Authorization;
    (function (Authorization) {
        var NG;
        (function (NG) {
            var Models;
            (function (Models) {
                var PostDataModel = (function () {
                    function PostDataModel(obj) {
                        if (obj != null) {
                            this.Id = obj.Id;
                            this.Name = obj.Name;
                            this.IsTrue = obj.IsTrue;
                            this.CreatedOn = obj.CreatedOn;
                        }
                    }
                    return PostDataModel;
                }());
                Models.PostDataModel = PostDataModel;
            })(Models = NG.Models || (NG.Models = {}));
        })(NG = Authorization.NG || (Authorization.NG = {}));
    })(Authorization = DeBiese.Authorization || (DeBiese.Authorization = {}));
})(DeBiese || (DeBiese = {}));

File: Sample Code/DeBiese.Authorization/DeBiese.Authorization.API/Models/PostData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DeBiese.Authorization.API.Models
{
    public class PostData
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool IsTrue { get; set; }
        public DateTime CreatedOn { get; set; }

        public override string ToString()
        {
            //String interpolation starting from .net 4.6
            return $"PostData with [Id: {this.Id.ToString()}] - [Name: {this.Name}] - [IsTrue: {this.IsTrue.ToString()}] - [CreatedOn: {this.CreatedOn.ToString("dd/MM/yyy")}].";
        }
    }
}